sourmash_lib/__init__.py

```diff
@@ -14,6 +14,8 @@
 except ImportError:
     pass
 
+DEFAULT_SEED=MinHash(1,1).seed
```

Comment:
yes, I think that's cleaner, but this will at least work for the moment. If I don't get to it on this PR I'll add an issue.